src/control/server/instance.go

```diff
@@ -58,17 +58,17 @@ type IOServerInstance struct {
 	msClient          *mgmtSvcClient
 	waitDrpc          atm.Bool
 	drpcReady         chan *srvpb.NotifyReadyReq
-	storageReady      chan struct{}
+	storageReady      chan bool
 	ready             atm.Bool
+	startChan         chan bool
```

Comment:
Why switch these to bool channels? The convention when a channel is used for signaling is to use the empty struct and just close the channel.

Reply:
both will need to be reused multiple times and I would prefer to do this with the Boolean channel.  May also want to return false in future.